.readthedocs.yml

```diff
@@ -13,7 +13,7 @@ formats: all
 
 # Optionally set the version of Python and requirements required to build your docs
 python:
-  version: 3.7
+  version: 3.8
```

Comment:
Should I insert a comment on why 3.8 is used?

Reply:
Yes, please. In the future, we may want to update Python version to 3.9 or so, and such a comment will help us.